byway/src/main/webapp/destinations.html

```diff
@@ -2,65 +2,68 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Select Destinations</title>
     <link rel="stylesheet" href="destinations.css" >
```

Comment:
You want the page specific destinations.css to take precedence, so you would place it after the shared script.css

Reply:
done